README.md

```diff
@@ -121,14 +121,44 @@ go build -o chainlink ./core/
 ```bash
 yarn
 yarn setup:contracts
+go generate ./...
+go run ./packr/main.go ./core/eth/
 ```
 
-3. Ready for testing:
+3. Generate and compile static assets:
+
+```bash
+go generate ./...
+go run ./packr/main.go ./core/eth/
+```
+
+4. Prepare your development environment:
+
+Create the test database:
+```bash
+createdb chainlink_test
+```
+
+Set your environment to allow for development and point to the test database:
+```bash
+export DATABASE_URL=postgresql://127.0.0.1:5432/chainlnk_test?sslmode=disable
+export CHAINLINK_DEV=true # I prefer to use direnv and skip this
+```
+
+5.  Run migrations on your test database:
```

Comment:
This drops/creates the DB for you

Reply:
@se3000 how do you feel about updating the title to something like `Drop/Create test database and run migrations` per the suggestion from @samsondav